toolkit/scripts/toolkit.mk

```diff
@@ -127,3 +127,18 @@ $(valid_arch_spec_names): $(go-specarchchecker) $(chroot_worker) $(local_specs)
 		--log-level=$(LOG_LEVEL) \
 		--log-file="$(valid_arch_spec_names_logs_path)" \
 		--log-color="$(LOG_COLOR)"
+
+##help:target:install-prereqs=Install build prerequisites automatically.
+install-prereqs:
```

Comment:
I'm fine with there being a make target here **too**, but can we have an even simpler entry point that doesn't even count on `make` being installed? In other words, if we placed this logic (including the distro detection) in a simple `install-prereqs.sh` script in an easy-to-find place in the repo, then it would be runnable with only `bash` and the OS's native package manager. That's a bit closer to what I'm accustomed to seeing in other projects.

Reply:
You are looking for something more than https://github.com/microsoft/azurelinux/pull/11081/files#diff-0f327d8103d75628337c8392aa0ad989a1e5e8d6967faa89b892c2a2c1b1295cR10 ?
I can do that if needed.